repository/src/main/resources/db/migration/V20230723_4590__CCD-4590_CreateCaseMessage.sql

```diff
@@ -0,0 +1,72 @@
+insert into field_type (created_at, reference, version, base_field_type_id)
+values (now(), 'CaseMessage', 1,
+(select id from field_type where reference = 'Complex'
+and jurisdiction_id is null
+and version = (select max(version)
+from field_type where reference = 'Complex'
+and jurisdiction_id is null
+and base_field_type_id is null))
+);
+
+insert into complex_field (reference, label, security_classification, field_type_id, complex_field_type_id)
+values ('caseMessageId', 'caseMessageId', 'PUBLIC',
+(select id from field_type where reference = 'Text' and version = 1 and jurisdiction_id is null),
+(select id from field_type where reference = 'CaseMessage' and version = 1 and jurisdiction_id is null));
+
+insert into complex_field (reference, label, security_classification, field_type_id, complex_field_type_id)
+values ('subject', 'subject', 'PUBLIC',
+(select id from field_type where reference = 'Text' and version = 1 and jurisdiction_id is null),
+(select id from field_type where reference = 'CaseMessage' and version = 1 and jurisdiction_id is null));
+
+insert into complex_field (reference, label, security_classification, field_type_id, complex_field_type_id)
+values ('body', 'body', 'PUBLIC',
+(select id from field_type where reference = 'Text' and version = 1 and jurisdiction_id is null),
+(select id from field_type where reference = 'CaseMessage' and version = 1 and jurisdiction_id is null));
+
+insert into field_type (created_at, reference, version, base_field_type_id, collection_field_type_id)
+values (now(), 'documentCollection', 1,
+    (select id from field_type where reference = 'Collection'
+        and jurisdiction_id is null
+        and version = (select max(version)
+        from field_type where reference = 'Collection'
+        and jurisdiction_id is null
+        and base_field_type_id is null)),
+    (select id from field_type where reference = 'document'
```

Comment:
```suggestion
    (select id from field_type where reference = 'Document'
```

Reply:
As per https://github.com/hmcts/ccd-definition-store-api/blob/master/repository/src/main/resources/db/migration/V0001__Base_version.sql#L2563-L2564 it's Document not document